apps/cowswap-frontend/src/modules/application/containers/App/RoutesApp.tsx

```diff
@@ -18,13 +18,14 @@ import { RedirectPathToSwapOnly, RedirectToPath } from 'legacy/pages/Swap/redire
 
 import { Routes as RoutesEnum, RoutesValues } from 'common/constants/routes'
 import Account, { AccountOverview } from 'pages/Account'
+import AdvancedOrdersPage from 'pages/AdvancedOrders'
 import AnySwapAffectedUsers from 'pages/error/AnySwapAffectedUsers'
 import { HooksPage } from 'pages/Hooks'
+import LimitOrderPage from 'pages/LimitOrders'
 import { SwapPage } from 'pages/Swap'
+import YieldPage from 'pages/Yield'
 
 // Async routes
-const LimitOrders = lazy(() => import(/* webpackChunkName: "limit_orders" */ 'pages/LimitOrders'))
```

Comment:
As it turned out, there is no difference with/without lazy-loading, bundle size is the same

Reply:
So, we make the app load worse for nothing?
Surely at least the initial download would be faster, no?

Reply:
As I understand, due to dependencies hell, limit orders module was already included in the main bundle and was additionally double-loaded as lazy module.
We must get rid of wrong dependencies to make the module truely lazy, but I don't have a capacity right now.

Reply:
p.s. the dependencies problem wasn't introduced in this PR